prometheus/plugin.go

```diff
@@ -46,7 +47,7 @@ func pluginTableDefinitions(ctx context.Context, p *plugin.TableMapData) (map[st
 	}
 
 	for _, i := range metricNames {
-		tableCtx := context.WithValue(ctx, "metric_name", i)
+		tableCtx := context.WithValue(ctx, ctxKey{}, i)
```

Comment:
@beudbeud What does this change do?

Reply:
@cbruno10 because for fix linting [error](https://github.com/turbot/steampipe-plugin-prometheus/pull/23/checks?check_run_id=17529852681)

Reply:
```
prometheus/plugin.go:48:38: SA1029: should not use built-in type string as key for value; define your own type to avoid collisions (staticcheck)
		tableCtx := context.WithValue(ctx, "metric_name", i)
```